server/src/test/java/org/elasticsearch/index/search/geo/GeoUtilsTests.java

```diff
@@ -386,6 +386,8 @@ public void testNormalizePointEdgeCases() {
         assertNormalizedPoint(new GeoPoint(180.0, 360.0), new GeoPoint(0.0, 180.0));
         assertNormalizedPoint(new GeoPoint(-90.0, -180.0), new GeoPoint(-90.0, -180.0));
         assertNormalizedPoint(new GeoPoint(90.0, 180.0), new GeoPoint(90.0, 180.0));
+        assertNormalizedPoint(new GeoPoint(100.0, 180.0), new GeoPoint(80.0, 0.0));
+        assertNormalizedPoint(new GeoPoint(100.0, -180.0), new GeoPoint(80.0, 0.0));
```

Comment:
should there be tests for `-100` latitudes?

Reply:
I think the problem here is not with latitude normalization, the problem is in longitudes, when latitude normalization also triggers longitude change. The issue was that `180.0` was correctly becoming `0.0` but `-180.0` was becoming `-360.0`. So, the latitude value normalization is tested somewhere else, and only the fact that normalization is triggered is important, but since it was causing questions I have added two cases with `-100.0` for completeness.